src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking-overview/checking-overview.component.scss

```diff
@@ -13,6 +13,12 @@
 }
 
 #add-question-button {
+  color: white;
```

Comment:
Set `color=primary` on the button, and none of this change to CSS rules should be needed.